Doc/library/sqlite3.rst

```diff
@@ -155,137 +155,9 @@ Reference
 Module functions and constants
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Maybe worth splitting this into two sections, one for functions and one for module-level constants, for ease of identification and navigation, especially since `UPPER_SNAKE_CASE` isn't used consistently to identify constants, and it makes the alphabetical ordering more obvious?